### PR TITLE
[TASK] Landing: Show register on first visit (RT-2982)

### PR DESCRIPTION
**Pull request by Mesrop**

### CODE REVIEW

**Comment by mrajvanshy** [comment]
When I try to open the PR environment and visit Ripple Trade on a browser that I've visited it on before, the browser keeps refreshing itself and the login page never loads

File: src/js/services/id.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ module.factory('rpId', ['$rootScope', '$location', '$route', '$routeParams', '$t
    * This creates the version of the username that is displayed in the UI.
    */
   Id.normalizeUsernameForDisplay = function (username) {
-    username = ""+username;
+    username = "" + username;
```
Comment (jublo) [inline src/js/services/id.js]: We prefer single quotes `'`.

File: src/js/services/id.js [hunk 2]
```diff
@@ -70,7 +70,7 @@ module.factory('rpId', ['$rootScope', '$location', '$route', '$routeParams', '$t
    * servers.
    */
   Id.normalizeUsernameForInternals = function (username) {
-    username = ""+username;
+    username = "" + username;
```
Comment (jublo) [inline src/js/services/id.js]: See above (I won't repeat it for all occurrences because I'm lazy).

File: src/js/services/id.js [hunk 3]
```diff
@@ -658,14 +639,19 @@ module.factory('rpId', ['$rootScope', '$location', '$route', '$routeParams', '$t
...
+      return;
     }
   });
 
   $scope.$on('$routeChangeStart', function (ev, next) {
     if (!$scope.loginStatus) {
       var tab = next.tabName;
-
-      if (tab !== 'register' && tab !== 'migrate' && tab !== 'recover' && tab !== '404' && tab !== 'privacypolicy' && tab !== 'tou') {
+      if (tab !== 'login' && tab !== 'migrate' && tab !== 'recover' && tab !== '404' && tab !== 'privacypolicy' && tab !== 'tou') {
```
Comment (jublo) [inline src/js/services/id.js]: This *might* be refactored to use an array.

**Comment by Mesrop** [comment]
@mrajvanshy @jublo can you please check again

**Comment by mrajvanshy** [comment]
LGTM

**Comment by MatthewPhinney** [comment]
LGTM

**Pull request merged**